commands/post.go

```diff
@@ -204,12 +206,13 @@ func postListCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 				post, err := eventDataToPost(event.GetData())
 				if err != nil {
 					fmt.Println("Error parsing incoming post: " + err.Error())
```

Comment:
I know this was part of the original code but could you please change this to `printer.PrintError`?

Reply:
Fixed it